docs/install/airgap.md

```diff
@@ -8,23 +8,23 @@ You can either deploy via the `rke2-airgap-images` tarball release artifact, or
 All files mentioned in the steps can be obtained from the assets of the desired released rke2 version [here](https://github.com/rancher/rke2/releases).
 
 If running on an air-gapped node with SELinux enabled, you must manually install the necessary SELinux policy RPM before performing these steps. See our [RPM Documentation](https://docs.rke2.io/install/methods/#rpm) to determine what you need.
- 
+
 If running on an air-gapped node running SELinux, CentOS, or RHEL 8, with SELinux enabled, the following are required dependencies when doing an [RPM install](https://docs.rke2.io/install/methods/#rpm):
```

Comment:
All the steps listed on this fie requires root user access, but since it is an advance topic I didn't thought we require to point it out. wdyt?

Reply:
Please update. We should try to keep things as consistent as possible.